### PR TITLE
fix(tests): Re-enable all runtime tests

### PR DESCRIPTION
**Pull request by janpio**
Seems we forgot a `slice(0,1)` in the past that disabled all but 1 runtime test.
This removes that code again and updates the snapshots to the current state (h/t @Jolg42).

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -144,7 +144,7 @@ jobs:
       # - name: Setup tmate session
       #   uses: mxschmitt/action-tmate@v3
 
-      - run: pnpm run jest integration/${{ matrix.database }} -- --forceExit --maxConcurrency=8
+      - run: pnpm run jest integration/${{ matrix.database }} -- --forceExit --maxConcurrency=8 --verbose
```
Comment (Jolg42) [inline .github/workflows/test.yml:147]: We can probably remove this
```suggestion
      - run: pnpm run jest integration/${{ matrix.database }} -- --forceExit --maxConcurrency=8
```

Comment (janpio) [reply .github/workflows/test.yml:147]: I added it to see which tests we were running. Only with that it was possible to see that this was only running 1 test vs. many as expected.

Comment (Jolg42) [reply .github/workflows/test.yml:147]: Since we removed `slice(0,1)` I don't think we need it though?

Comment (Jolg42) [reply .github/workflows/test.yml:147]: It does help indeed

**Comment by Jolg42** [comment]
Thanks for the util @williamluke4 from https://github.com/prisma/prisma/pull/6059 💯

**Comment by Jolg42** [comment]
Some MySQL fun to fix because we map `serial` to BigInt in MySQL
https://www.prisma.io/docs/concepts/database-connectors/mysql#prisma-migrate

**Comment by Jolg42** [comment]
@williamluke4 NAPI tests are failing 
SQLite with https://github.com/prisma/prisma/pull/6066/checks?check_run_id=2498970379#step:8:682
```
Bus error (core dumped)
 ERROR  Command failed with exit code 135.
```
PostgreSQL (and MySQL, mariadb, mssql) https://github.com/prisma/prisma/pull/6066/checks?check_run_id=2498970412#step:8:695
```
Illegal instruction (core dumped)
 ERROR  Command failed with exit code 132.
```

Also on my machine running when running `env PRISMA_FORCE_NAPI='true' env DEBUG="*" jest integration.sqlite --verbose` tests are all green and I can't reproduce.
One thing I can notice is that `src/__tests__/integration/sqlite/runtime.test.ts` is quite slow (a lot slower than without NAPI env var 🤔)

**Review by Jolg42 — APPROVED** [review]
Thanks Will!

**Pull request merged**